### ISSUE TITLE
Import auto-complete in typescript vscode

### ISSUE THREAD

**Issue by gamegee** [codota/tabnine-vscode#102]
**please complete the following information:**	
 - OS version: Mac Catalina]
 - Editor: vscode
 - Editor version: 1.45.0	
 - Programming language: ts
 - TabNine extension version: 2.8.2

**Issue Details:**

It is more a question than an Issue actually.
I just installed TabNine and I was wondering if it could import other .ts files with autocompletion. (like IntelliSense is doing).

E.g. (Lets say I have 2 files)
```tsx
// Foo.ts
export const Foo = () => {}


// Bar.ts - BEFORE AUTO-SUGGESTION
const Bar = () => {
  Foo // auto-suggestion here
}

// Bar.ts - AFTER AUTO-SUGGESTION
import { Foo } from './Foo.ts'
const Bar = () => {
  Foo()
}
```

**Comment by dimacodota** [comment]
Great! [here](https://github.com/codota/tabnine-vscode/releases/tag/auto-import-beta) is the beta version (please download the vsix, and then you can install it in vscode),  we appreciate all the feedback on the real daily use of the auto import feature.

**Comment by dimacodota** [comment]
The auto import support was released in the 2.8.4 version. 
To enable it, go to settings and set the `Tabnine: Experimental Auto Imports` to true.

**Comment by gamegee** [comment]
@dimacodota 
Thanks for sharing Auto Import, appreciated. I am a recent user of Tabnine, and it's really cool !

After few weeks using the `Auto Imports` I can give you some feedback about my experience

- There is no notification of imports, the TabNine autocomplete can sometime trigger an import but I don't really know when, and usually I only notice the import by scrolling on top of my js/ts file. Unlike intellisense, there is no specific `import` icon near the autosuggestion dropdown (it could help to know when import occur).   

**Tabnine suggestion that trigger import**  

![image](https://user-images.githubusercontent.com/15988848/83811094-7298d280-a6b9-11ea-9030-e44c85dea064.png)

**Intellisense suggestion that trigger import**
With Message + specific icon
![image](https://user-images.githubusercontent.com/15988848/83811171-93f9be80-a6b9-11ea-9744-896bb7e0fa10.png)



- Because TabNine doesn't know about project structure/config (I guess?), it will sometimes trigger wrong import for global scoped variable (It is an edge case related to my project)
```tsx
// I have a global type Identifier in a global.d.ts file

// But writing this:
type user = {
  id: Identifier
}
// Will trigger this import
import { Identifier } from 'typescript'
```

- Sometimes TabNine will trigger multiple import resulting in something like this:
```tsx
import { Foo, Foo } from './bar'
```
- Default export (aka non-named exports) can't be auto imported
```tsx
// components/Button.tsx
const Button = () => <button>hello</button>
export default Button

// components/OtherFile.tsx
...
// Writting this will trigger autocomplete but no auto-import
<Button>
```

These are the main UX improvements I noticed for now
